<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expenses</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(172, 125, 136);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(248, 236, 209); 
font: 700 italic 13pt &quot;Garamond&quot;</string>
   </property>
   <property name="title">
    <string>  Expenses  </string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(248, 236, 209); 
font: 700 italic 11pt &quot;Garamond&quot;</string>
    </property>
    <property name="title">
     <string>  Estimated Cost of Face-to-Face Daily Expences   </string>
    </property>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>Transportation:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbTranspo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>Meal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>Masks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>Sanitizers:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbMeal">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbMasks">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbSan">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbFS">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>Face Shields:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(248, 236, 209);
font: 8pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>ENTER COSTS IN PHP</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(248, 236, 209); 
font: 700 italic 11pt &quot;Garamond&quot;</string>
    </property>
    <property name="title">
     <string>  Estimated Cost of Online Monthly Expences  </string>
    </property>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>Electricity Bill:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbEB">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>Internet Bill:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbIB">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(248, 236, 209);
font: 8pt &quot;Impact&quot;;</string>
     </property>
     <property name="text">
      <string>ENTER COSTS IN PHP</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
    </property>
    <property name="text">
     <string>Student ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;
</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonSearch">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>SEARCH RECORDS:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboRecord">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>EDIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDelete">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAdd">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>Total Cost of Face-to-Face Monthly Expences: (x30)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbTC1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;
color: rgb(248, 236, 209);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>Total Cost of Online Monthly Expences:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbTC2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(248, 236, 209);
font: 9pt &quot;Impact&quot;;
color: rgb(248, 236, 209);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 182, 171);
font: 10pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>FILL IN THE BOXES IN CAPITAL LETTERS. THANKYOU!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCompute">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 236, 209);
color: rgb(128, 80, 42);
font: 700  9pt &quot;Garamond&quot;</string>
   </property>
   <property name="text">
    <string>COMPUTE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
